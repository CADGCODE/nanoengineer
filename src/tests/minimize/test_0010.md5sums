stdout B7FEACE00C3034E294980C2459521DB0
stderr D41D8CD98F00B204E9800998ECF8427E
test_0010.xyz BC678F3AAB2C86A9E9A62A4CF55F68AD
