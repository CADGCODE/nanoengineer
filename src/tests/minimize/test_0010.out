======= test_0010.test =======
======= exitvalue =======
0
======= stderr =======
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_0010.mmp  > test_0010.xyz

Total Ke = 4.968000e-20
final RMS gradient=0.857123 after 13 iterations
======= test_0010.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_0010.mmp
# Output File: test_0010.xyz
# Trace File: test_0010.trc
# Number of Atoms: 8
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 1 columns:
# name: torque (nn-nm)
#

# rms force, high force
1 689.52 1052.14
2 328.50 339.90
3 178.64 284.18
4 120.00 166.61
5 119.61 215.29
6 45.96 66.56
# Switching to Conjugate-Gradient
7 25.15 31.11
8 18.91 21.34
9 8.88 17.14
10 3.56 4.76
11 2.19 3.01
12 0.86 1.28
13 0.86 1.28
# Done: Minimization final rms: 0.857123, highForce: 1.276734
======= test_0010.xyz =======
8
RMS=0.857123
C -0.436136 -4.013652 0.010303
H -1.068352 -3.395732 -0.621955
H -1.068296 -4.645794 0.628407
H 0.181882 -4.645808 -0.621913
C 0.458207 -3.119276 0.904543
H 1.090360 -3.737303 1.536676
H -0.159916 -2.487183 1.536635
H 1.090251 -2.487252 0.286304
